crates/router/src/routes/app.rs

```diff
@@ -1603,6 +1603,7 @@ impl User {
                     .route(web::get().to(list_merchants_for_user)),
             )
             .service(web::resource("/permission_info").route(web::get().to(get_authorization_info)))
+            .service(web::resource("/roles_info").route(web::get().to(get_role_information)))
```

Comment:
Why did the route change?

Reply:
But permission_info was also giving multiple. Can you evaluate this and check if this is the right practice.